<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IMUWidget</class>
 <widget class="QWidget" name="IMUWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>878</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Data</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="dataTree">
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="positionGroupBox">
     <property name="title">
      <string>Position</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="xPosLabel">
        <property name="text">
         <string>X:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="xPos">
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="minimum">
         <double>-10000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="yPosLabel">
        <property name="text">
         <string>Y:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="yPos">
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="minimum">
         <double>-10000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>10.000000000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="zPosLabel">
        <property name="text">
         <string>Z:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="zPos">
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="minimum">
         <double>-10000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="attitudeGroupBox">
     <property name="title">
      <string>Attitude</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="rollLabel">
        <property name="text">
         <string>Roll:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="roll">
        <property name="minimum">
         <double>-180.000000000000000</double>
        </property>
        <property name="maximum">
         <double>180.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="pitchLabel">
        <property name="text">
         <string>Pitch:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="pitch">
        <property name="minimum">
         <double>-180.000000000000000</double>
        </property>
        <property name="maximum">
         <double>180.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="yawLabel">
        <property name="text">
         <string>Yaw:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="yaw">
        <property name="minimum">
         <double>-180.000000000000000</double>
        </property>
        <property name="maximum">
         <double>180.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="localAttitudeGroupBox">
     <property name="title">
      <string>Local attitude</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="rollLabel_2">
        <property name="text">
         <string>Roll:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="localRollLabel">
        <property name="text">
         <string>roll</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="pitchLabel_2">
        <property name="text">
         <string>Pitch:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="localPitchLabel">
        <property name="text">
         <string>pitch</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="yawLabel_2">
        <property name="text">
         <string>Yaw:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="localYawLabel">
        <property name="text">
         <string>yaw</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="stickGroupBox">
     <property name="title">
      <string>Stick</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QFormLayout" name="colorFormLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Spheres color:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QToolButton" name="sColorToolButton">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Connections color:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QToolButton" name="cColorToolButton">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="radiusFormLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Spheres radius:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="sRadiusDoubleSpinBox">
          <property name="decimals">
           <number>0</number>
          </property>
          <property name="minimum">
           <double>20.000000000000000</double>
          </property>
          <property name="maximum">
           <double>150.000000000000000</double>
          </property>
          <property name="value">
           <double>60.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Connections radius:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="cRadiusDoubleSpinBox">
          <property name="decimals">
           <number>0</number>
          </property>
          <property name="minimum">
           <double>5.000000000000000</double>
          </property>
          <property name="maximum">
           <double>60.000000000000000</double>
          </property>
          <property name="value">
           <double>20.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="showCheckBox">
     <property name="text">
      <string>Show</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cRadiusDoubleSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>IMUWidget</receiver>
   <slot>updateConnectionsView()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>652</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pitch</sender>
   <signal>valueChanged(double)</signal>
   <receiver>IMUWidget</receiver>
   <slot>updateAttitude()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>566</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>roll</sender>
   <signal>valueChanged(double)</signal>
   <receiver>IMUWidget</receiver>
   <slot>updateAttitude()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>566</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sRadiusDoubleSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>IMUWidget</receiver>
   <slot>updateSpheresView()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>626</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>IMUWidget</receiver>
   <slot>switchVisibility(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>688</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xPos</sender>
   <signal>valueChanged(double)</signal>
   <receiver>IMUWidget</receiver>
   <slot>updatePosition()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>507</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yPos</sender>
   <signal>valueChanged(double)</signal>
   <receiver>IMUWidget</receiver>
   <slot>updatePosition()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>507</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yaw</sender>
   <signal>valueChanged(double)</signal>
   <receiver>IMUWidget</receiver>
   <slot>updateAttitude()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>566</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zPos</sender>
   <signal>valueChanged(double)</signal>
   <receiver>IMUWidget</receiver>
   <slot>updatePosition()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>507</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateSpheresView()</slot>
  <slot>updateConnectionsView()</slot>
  <slot>updatePosition()</slot>
  <slot>updateAttitude()</slot>
  <slot>switchVisibility(bool)</slot>
 </slots>
</ui>
